### PR TITLE
Retrieval of Products and Updates to use version 1.1.0 of cwms-http-client (Allowing for POST Requests)

### PR DESCRIPTION
**Pull request by rma-bryson**
Retrieval of Products

### CODE REVIEW

File: cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/ProductsEndpointInput.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    private final String productId;
+    /**
+     * Endpoint Input for Products to handle input parameters.
+     *
+     * @param productId - ID of product (or null if retrieving all products)
+     */
+
+    public ProductsEndpointInput(String productId) {
+        this.productId = Objects.requireNonNull(productId, "Cannot access acquirable file without acquirable id");
```
Comment (rma-psmorris) [inline cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/ProductsEndpointInput.java]: aquirable file in product endpoint? Does this API download files?

Comment (rma-bryson) [reply cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/ProductsEndpointInput.java]: That's a typo that I missed. I'll fix it.

Comment (rma-bryson) [reply cumulus-client/src/main/java/mil/army/usace/hec/cumulus/client/controllers/ProductsEndpointInput.java]: fixed it

**Review by rma-psmorris — APPROVED** [review]
good to go

**Pull request merged**